src/system_tests/mhd_system_tests.cpp

```diff
@@ -32,15 +33,15 @@ class tMHDSYSTEMLinearWavesParameterizedAngle : public ::testing::TestWithParam<
 
  protected:
   systemTest::SystemTestRunner waveTest;
+  inline static std::unordered_map<std::string, double> n32_l2norms;
```

Comment:
You might want to rename this to something like "`low_res_l2norms`". I wasn't sure what the `n32` prefix or the "Check the scaling" comments in the higher-resolution tests meant, so this might clarify those.

Reply:
Good point. Fixed in commit [166a102](https://github.com/cholla-hydro/cholla/pull/305/commits/166a10230a6eb8d90a24fb13fb702d7ee2114ddb).